lib/iris/mesh/components.py

```diff
@@ -3017,6 +2986,70 @@ def summary(self, *args, **kwargs):
             result = "\n".join(lines)
         return result
 
+    def _load_points_and_bounds(self):
```

Comment:
All references to `location` need to be `self.location` or this won't be able to run.

Reply:
Ditto `axis` -> `self.axis`.